Jenkinsfile

```diff
@@ -78,6 +78,22 @@ def runCleanupResource(String backend){
     """
 }
 
+def runRestoringMonitoringStack(){
+    sh """
+    #!/bin/bash
+
+    set -xe
+    env
+
+    export MONITOR_STACK_TEST_ID=`cat 20200121-083324-443210/test_id`
```

Comment:
I guess you didn't want to hardcode it to specific folder..

Reply:
yes, you right.